vpn/src/main/java/com/duckduckgo/mobile/android/vpn/pixels/DeviceShieldPixelNames.kt

```diff
@@ -36,6 +36,7 @@ enum class DeviceShieldPixelNames(override val pixelName: String, val enqueue: B
     ATP_DISABLE_UPON_APP_LAUNCH("m_atp_ev_disabled_on_launch_c"),
     ATP_DISABLE_UPON_APP_LAUNCH_DAILY("m_atp_ev_disabled_on_launch_d"),
     ATP_ENABLE_DAILY("m_atp_ev_enabled_d"),
+    ATP_DISABLE("m_atp_ev_disabled_c"),
```

Comment:
I would call this `ATP_UPTIME("m_atp_ev_uptime_c")`. We already have a `ATP_DISABLE_DAILY` and the convention here is that `ATP_DISABLE_DAILY` would be sent along with `ATP_DISABLE` which can't happen. So it's a bit confusing